src/sonic-host-services/scripts/caclmgrd

```diff
@@ -275,20 +275,24 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
         return fwd_snmp_traffic_from_namespace_to_host_cmds
 
-    def is_rule_ipv4(self, rule_props, ethertype):
+    def get_ethertype_from_rule_props(rule_props):
+        ethertype = int(rule_props['ETHER_TYPE'], 0)
```

Comment:
Suggest following my example and using

```python
    ethertype_str = rule_props.get('ETHER_TYPE', '0')
    ethertype = int(ethertype_str, 0)
```

Because this will gracefully handle the case where `ETHER_TYPE` isn't present in `rule_props` and return `0`. `ethertype = int(rule_props['ETHER_TYPE'], 0)` will crash in this case.

Reply:
Got it. Will change this. Thanks.